exercicios/para-casa/entregas/ellen-dutra/ContaBancaria.test.js

```diff
@@ -0,0 +1,40 @@
+const ContaBancaria = require('./ContaBancaria');
+
+describe('Teste de ContaBancaria', () => {
+  let conta;
+  beforeEach(() => {
+    conta = new ContaBancaria('Ellen', 1500, 1000);
+  });
+
+  test('Consulta o saldo', () => {
+    expect(conta.consultarSaldo()).toEqual(1500);
+  });
+
+  test('Saque na conta caso haja saldo suficiente', () => {
+    expect(conta.efetuarSaque(1000)).toEqual(500);
+  });
+
+  test('Saque na conta com saldo insuficiente, mas com limite disponível', () => {
+    expect(conta.efetuarSaque(700)).toEqual(800);
+    expect(conta.efetuarSaque(900)).toEqual(-100);
+  });
+
+  test('Não permite saque na conta sem saldo e limite suficientes', () => {
+    expect(() => {
+      conta.efetuarSaque(3000);
+    }).toThrow();
```

Comment:
Ao testar se uma exceção é lançada, é útil adicionar uma mensagem de erro descritiva para explicar o que está sendo testado. Isso facilita a depuração, caso um teste falhe.

```suggestion
    }).toThrow('Saldo e limite insuficientes');
```